app/web/src/i18n.ts

```diff
@@ -16,7 +18,7 @@ i18n
           });
         return;
       }
-      import(`./features/locales/${language}.json`)
+      import(`./resources/locales/${language}.json`)
```

Comment:
Why resources? Seems a bit odd to have only global strings there and others in the respective feature folder.

Reply:
I started adding the constants in some components (under the `components` folder) into the global namespace, so it doesn't belong to `features` exclusively anymore. The `resources` folder feels "global-ish" to me based on what we got in there so far.